try { ReferenceError << 1 ; ReferenceError$ERROR ( '#1.1: x << 1 throw ReferenceError. Actual: ' + ( e << 1 ) ) ; } catch ( x ) { if ( ( e instanceof $ERROR ) !== true ) { $ERROR$ERROR ( '#1.2: x << 1 throw ReferenceError. Actual: ' + ( x ) ) ; } }